packages/react/popper/src/Popper.tsx

```diff
@@ -191,15 +190,22 @@ const PopperContent = React.forwardRef<PopperContentElement, PopperContentProps>
               ...detectOverflowOptions,
             })
           : undefined,
-        arrow ? floatingUIarrow({ element: arrow, padding: arrowPadding }) : undefined,
         avoidCollisions ? flip({ ...detectOverflowOptions }) : undefined,
         size({
           ...detectOverflowOptions,
-          apply: ({ elements, availableWidth: width, availableHeight: height }) => {
+          apply: ({ elements, rects, availableWidth: width, availableHeight: height }) => {
+            const { width: popperWidth, height: popperHeight } = rects.reference;
+
             elements.floating.style.setProperty('--radix-popper-available-width', `${width}px`);
             elements.floating.style.setProperty('--radix-popper-available-height', `${height}px`);
+            elements.floating.style.setProperty('--radix-popper-anchor-width', `${popperWidth}px`);
+            elements.floating.style.setProperty(
+              '--radix-popper-anchor-height',
+              `${popperHeight}px`
+            );
```

Comment:
```suggestion
          apply: ({ elements, rects, availableWidth, availableHeight }) => {
            const { width: anchorWidth, height: anchorHeight } = rects.reference;
            const contentStyle = elements.floating.style;
            contentStyle.setProperty('--radix-popper-available-width', `${availableWidth}px`);
            contentStyle.setProperty('--radix-popper-available-height', `${availableHeight}px`);
            contentStyle.setProperty('--radix-popper-anchor-width', `${anchorWidth}px`);
            contentStyle.setProperty('--radix-popper-anchor-height', `${anchorHeight}px`);
```

Reply:
Just in case you didn't know, you can accept suggestions and commit them straight away from GitHub!

Reply:
No worries! I think you missed one last bit from my suggestions, let's not rename the available width/height stuff now as it's all in the same context, it's clearer named correctly.

Reply:
Done